src/Raven.Server/Config/Categories/SecurityConfiguration.cs

```diff
@@ -172,8 +172,7 @@ public class SecurityConfiguration : ConfigurationCategory
 
         public bool IsCertificateConfigured =>
             string.IsNullOrWhiteSpace(CertificatePath) == false ||
-            string.IsNullOrWhiteSpace(CertificateLoadExec) == false ||
-            string.IsNullOrWhiteSpace(CertificateExec) == false;
```

Comment:
What about backward compatibility?

Reply:
There is no backward compatibility in regards to this configuration property.
I don't remember why but we gave it up when we did this:
https://github.com/ravendb/ravendb/blob/v4.2/src/Raven.Server/RavenServer.cs#L1356

It's funny because that commit's title is "backward compatability" :) 
https://github.com/ravendb/ravendb/pull/9041/commits/09dcccec1999fd077bf6181e3411152d30c4b1c7